rllib/utils/replay_buffers/episode_replay_buffer.py

```diff
@@ -208,17 +233,14 @@ def sample(
 
         B = 0
         T = 0
-        idx_cursor = 0
         while B < batch_size_B:
-            # Ran out of uniform samples -> Sample new set.
-            if len(index_tuples_idx) <= idx_cursor:
-                index_tuples_idx.extend(list(np.random.randint(
-                    0, len(self._indices), size=batch_size_B * 10
-                )))
+            # Pull a new uniform random index tuple: (eps_idx, ts_in_eps_idx).
+            index_tuple = self._indices[self.rng.integers(len(self._indices))]
```

Comment:
Just out of curiosity, why not use np.random.randint?
I have never dealt with np.random.default_rng so I was surprised to see it.

Reply:
Ah, great question! It was mentioned in the new numpy docs that this is what should be used going forward.